pkg/controller/utils/shared_utils.go

```diff
@@ -6,23 +6,26 @@
 package utils
 
 import (
-	"fmt"
-
 	datadoghqv1alpha1 "github.com/DataDog/datadog-operator/pkg/apis/datadoghq/v1alpha1"
 )
 
-// GetAPIKeySecretName return API key secret name
+// GetAPIKeySecretName return the API key secret name
 func GetAPIKeySecretName(dda *datadoghqv1alpha1.DatadogAgent) string {
 	if dda.Spec.Credentials.APIKeyExistingSecret != "" {
 		return dda.Spec.Credentials.APIKeyExistingSecret
 	}
 	return dda.Name
 }
 
-// GetAppKeySecretName return APP key secret name
+// GetAppKeySecretName return the APP key secret name
 func GetAppKeySecretName(dda *datadoghqv1alpha1.DatadogAgent) string {
 	if dda.Spec.Credentials.AppKeyExistingSecret != "" {
 		return dda.Spec.Credentials.AppKeyExistingSecret
 	}
-	return fmt.Sprintf("%s-%s", dda.Name, datadoghqv1alpha1.DefaultClusterAgentResourceSuffix)
+	return dda.Name
+}
+
+// GetTokenSecretName returns the token secret name
+func GetAuthTokenSecretName(dda *datadoghqv1alpha1.DatadogAgent) string {
```

Comment:
why this is a public function if it's not imported by other pkgs?

Reply:
Fixed!